pandas/io/formats/style.py

```diff
@@ -623,6 +623,40 @@ def applymap(self, func, subset=None, **kwargs):
                            (func, subset), kwargs))
         return self
 
+    def where(self, func, style1, style2=None, subset=None, **kwargs):
+        """
+        Apply a function elementwise, updating the HTML
+        representation with a style which is selected in
+        accordance with the return value of a function.
+
+        .. versionadded:: 0.21.0
+
+        Parameters
+        ----------
+        func : callable
```

Comment:
this should mimic the signature of ``DataFrame.where`` as much as possible

func -> cond
style1 -> value
style2 -> other